packages/email-components/src/components/Text/_theme.scss

```diff
@@ -11,6 +11,13 @@ $colors: (
 );
 
 $sizes: (
+    xsmall: (
+        mobile: (
+            font-size: email-tokens.$text-xsmall-font-size,
+            line-height: email-tokens.$text-xsmall-line-height,
+        ),
+        // xsmall variant has only one font size.
```

Comment:
Is this comment necessary?

Reply:
We wrote a similar comment here for variant small because all other variants have also a tablet definition.

Reply:
I see but it is obvious by missing definition for other breakpoints.